packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "16.0.0",
+  "version": "16.1.0",
```

Comment:
I'm doing a minor core bump here assuming we will release after this PR is merged. I'm making this minor because we have a PR merged that is a minor update for this release.